pandas/core/indexes/datetimes.py

```diff
@@ -1753,7 +1753,32 @@ def freq(self, value):
     is_leap_year = _field_accessor(
         'is_leap_year',
         'is_leap_year',
-        "Logical indicating if the date belongs to a leap year")
+        """
+        Return a boolean indicating if the date belongs to a leap year.
+
+        A leap year is a year, occurring every four years, which has 366 days
+        (instead of 365) including 29th of February as an intercalary day.
+
+        Returns
+        -------
+        is_leap_year : Series of boolean
```

Comment:
There's no need to name the return value, only it's type. And it's missing the description.

> The documentation of the return is also similar to the parameters. But in this case, no name will be provided, unless the method returns or yields more than one value (a tuple of values).

> The parameters are defined by their name, followed by a space, a colon, another space, and the type (or types). Note that the space between the name and the colon is important. Types are not defined for *args and **kwargs, but must be defined for all other parameters. After the parameter definition, it is required to have a line with the parameter description, which is indented, and can have multiple lines. The description must start with a capital letter, and finish with a dot.